.ci.yaml

```diff
@@ -24,12 +24,14 @@ targets:
     recipe: cocoon/cocoon
     properties:
       add_recipes_cq: "true"
+      bot_type: "regular"
```

Comment:
I don't think this is the approach we should go with. By default 99.9% of tasks are going to run on regular bots.

To get this working, you would need to update every ci.yaml and every flutter/infra starlark config

Reply:
Right I am not sure if this is the optimal approach. This is an implementation based on https://chrome-internal-review.googlesource.com/c/infradata/config/+/4685734/7 . The gerrit commit has been stale for a while and I am not sure if there are better approaches.

I started a discussion on this approach in the design doc at https://docs.google.com/document/d/1EacFm_abJIzRj95J92f_VyHwvG_d0HWlQRsxAUSp4yc/edit?resourcekey=0-fA7GHZRgrN1eH6hfepveJg&disco=AAAAYxIzxwc. Would be great if people could drop some thoughts. Thank you!

Reply:
update: updated to top level properties